From: Arthur Gautier <superbaloo+debian@superbaloo.net>
Date: Tue, 28 Feb 2012 18:50:07 +0100
Subject: Fix-init

---
 bin/init.d/shinken |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/bin/init.d/shinken
+++ b/bin/init.d/shinken
@@ -419,7 +419,9 @@
     statusoutput=$(do_status "$1")
     rc=$?
     if [ $rc -ne 0 ]; then
-        failuremsg="Couldn't get status of $1: $statusoutput"
+        failuremsg="Couldn't get status of $1 : Assuming already stopped"
+        log_success_msg "$failuremsg"
+        rc=0
     else
         stopoutput=$(do_stop "$1" 2>&1)
         rc=$?
