From: Arthur Gautier <aga@zenexity.com>
Date: Tue, 28 Feb 2012 21:56:02 +0100
Subject: Use nagios user

---
 bin/default/shinken.in |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bin/default/shinken.in b/bin/default/shinken.in
index f73ec13..6122f4e 100644
--- a/bin/default/shinken.in
+++ b/bin/default/shinken.in
@@ -143,5 +143,5 @@ SKONFDEBUGFILE="$LOG/skonf-debug.log"
 NICENESS=5
 
 # user/group for the var/run rights
-SHINKENUSER=shinken
-SHINKENGROUP=shinken
+SHINKENUSER=nagios
+SHINKENGROUP=nagios
