From: Arthur Gautier <superbaloo@superbaloo.net>
Date: Tue, 8 Jan 2013 11:12:37 +0100
Subject: Do not start shinken by default

---
 bin/default/shinken.in |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/bin/default/shinken.in b/bin/default/shinken.in
index 6122f4e..98295db 100644
--- a/bin/default/shinken.in
+++ b/bin/default/shinken.in
@@ -145,3 +145,9 @@ NICENESS=5
 # user/group for the var/run rights
 SHINKENUSER=nagios
 SHINKENGROUP=nagios
+
+# Kill switch
+# Comment out this line to enable shinken
+exit 99
+
+
