From: Arthur Gautier <superbaloo@superbaloo.net>
Date: Tue, 8 Jan 2013 17:31:36 +0100
Subject: No need for arbiter to log into workdir

---
 etc/nagios.cfg |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/nagios.cfg b/etc/nagios.cfg
index 4015206..c0381e9 100644
--- a/etc/nagios.cfg
+++ b/etc/nagios.cfg
@@ -151,7 +151,7 @@ human_timestamp_log=0
 
 
 # The arbiter can have it's own local log
-local_log=arbiterd.log
+#local_log=arbiterd.log
 
 # Set to 0 will enable the nagios behavior that will
 # launch event handlers even if the element is in downtime.
